src/phoenix/server/api/schema.py

```diff
@@ -224,7 +227,26 @@ def hdbscan_clustering(
 
 
 @strawberry.type
-class Mutation(ExportEventsMutation): ...
+class Mutation(ExportEventsMutation):
+    @strawberry.mutation
+    def delete_project(self, info: Info[Context, None], id: GlobalID) -> Query:
+        if (traces := info.context.traces) is None:
+            return Query()
+        type_name, node_id = from_global_id(str(id))
+        if type_name != "Project":
+            return Query()
+        traces.archive_project(node_id)
+        return Query()
+
+    @strawberry.mutation
+    def archive_project(self, info: Info[Context, None], id: GlobalID) -> Query:
+        if (traces := info.context.traces) is None:
+            return Query()
+        type_name, node_id = from_global_id(str(id))
+        if type_name != "Project":
+            return Query()
```

Comment:
should probably throw. The way we do this in the app server is we have a wrapper for from_global_id that lists the expected type. If it doesn't comply we throw as this means we have a cross tenancy attack